otter/utils.py

```diff
@@ -280,9 +282,33 @@ class log level.
         logger.addHandler(handler)
         if cls._socket_handler:
             logger.addHandler(cls._socket_handler)
+        if cls._queue_handler:
+            logger.addHandler(cls._queue_handler)
+            logger.setLevel(logging.INFO)
```

Comment:
This shouldn't alter the logging level since it's controller by a CLI flag and this will disrupt it. If your code is calling Otter via the CLI, use the `-v` flag to set the log level to INFO, or if it's calling otter as a library, have your code set the level of with `loggers.set_level` as in https://github.com/ucbds-infra/otter-grader/blob/master/otter/cli.py#L31.

Reply:
OK yes, I didn't like that moment. I am with you